src/module/rules/rule-element/crafting-ability.ts

```diff
@@ -61,25 +61,45 @@ class CraftingAbilityRuleElement extends RuleElementPF2e<CraftingAbilityRuleSche
         };
     }
 
-    override beforePrepareData(): void {
+    override onApplyActiveEffects(): void {
         if (this.ignored) return;
 
         const slug = this.resolveInjectedProperties(this.slug);
-        this.actor.system.crafting.entries[sluggify(slug, { camel: "dromedary" })] = {
-            slug,
-            label: this.label,
-            isAlchemical: this.isAlchemical,
-            isDailyPrep: this.isDailyPrep,
-            isPrepared: this.isPrepared,
-            batchSizes: this.batchSizes,
-            craftableItems: this.craftableItems,
-            maxItemLevel: Number(this.resolveValue(this.maxItemLevel)) || this.actor.level,
-            maxSlots: this.maxSlots,
-            preparedFormulaData: this.prepared,
-        };
+        const key = sluggify(slug, { camel: "dromedary" });
+        const maxSlots = this.maxSlots !== undefined ? Number(this.resolveValue(this.maxSlots)) : null;
+        const maxItemLevel = Number(this.resolveValue(this.maxItemLevel));
+        const existing = this.actor.system.crafting.entries[key];
```

Comment:
Nooo entries

Reply:
We can't kill the entries till we migrate. AELikes unfortunately still target that.